notebooks/official/model_evaluation/custom_tabular_classification_model_evaluation.ipynb

```diff
@@ -166,210 +175,158 @@
+        "Learn more about [setting up a project and a development environment](https://cloud.google.com/vertex-ai/docs/start/cloud-environment)."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "ce6043da7b33"
+        "id": "Nqwi-5ufWp_B"
       },
       "outputs": [],
       "source": [
-        "# ! gcloud auth login"
+        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "LOCATION = \"us-central1\"  # @param {type:\"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "0367eac06a10"
+        "id": "zgPO1eR3CYjk"
       },
       "source": [
-        "**3. Colab, uncomment and run:**"
+        "### Create a Cloud Storage bucket\n",
+        "\n",
+        "Create a storage bucket to store intermediate artifacts such as datasets."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "21ad4dbb4a61"
+        "id": "MzGDU7TWdts_"
       },
       "outputs": [],
       "source": [
-        "# from google.colab import auth\n",
-        "# auth.authenticate_user()"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "c13224697bfb"
-      },
-      "source": [
-        "**4. Service account or other**\n",
-        "* See how to grant Cloud Storage permissions to your service account at https://cloud.google.com/storage/docs/gsutil/commands/iam#ch-examples."
+        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "zgPO1eR3CYjk"
+        "id": "-EcIXiGsCePi"
       },
       "source": [
-        "### Create a Cloud Storage bucket\n",
-        "\n",
-        "Create a storage bucket to store intermediate artifacts such as datasets."
+        "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "MzGDU7TWdts_"
+        "id": "NIq7R4HZCfIc"
       },
       "outputs": [],
       "source": [
-        "BUCKET_URI = \"gs://your-bucket-name-unique\"  # @param {type:\"string\"}"
+        "! gsutil mb -l $LOCATION -p $PROJECT_ID $BUCKET_URI"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "-EcIXiGsCePi"
+        "id": "init_aip:mbsdk,all"
       },
       "source": [
-        "**Only if your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
+        "### Initialize Vertex AI SDK for Python\n",
+        "\n",
+        "To get started using Vertex AI, you must have an existing Google Cloud project and [enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com). \n",
```

Comment:
Change link text. Move "enable the" out of the link text:
- "you must have an existing Google Cloud project and enable the [Vertex AI API] ..."